src/LeagueOAuth2ServerBundle.php

```diff
@@ -38,7 +40,9 @@ private function configureSecurityExtension(ContainerBuilder $container): void
     {
         /** @var SecurityExtension $extension */
         $extension = $container->getExtension('security');
-        $extension->addSecurityListenerFactory(new OAuth2Factory());
+
+        // BC Layer for < 5.1 versions
```

Comment:
should we just drop support for < 5.1 ? this bundle isn't tagged, i use it in an application, but it's already in symfony 5 so i have no problem with dropping support for 4

Reply:
cc @chalasr